ui/kotlinx-coroutines-android/animation-app/gradle.properties

```diff
@@ -18,6 +18,6 @@ org.gradle.jvmargs=-Xmx1536m
 
 kotlin.coroutines=enable
 
-kotlin_version = 1.2.21
```

Comment:
this actually not used to define version of library used. We pick library version from Build.gradle file from top project kotlinx-coroutines-android.
Is that intentional?
Dependencies usually are not defined in gradle.properties, but in Build.gradle with other build configuration.